pkg/translator/testdata/collector_config/removeattributeprocessor_defaults.yaml

```diff
@@ -0,0 +1,31 @@
+receivers:
+    otlp/otlp_in:
+        protocols:
+            grpc:
+                endpoint: ${STRAWS_COLLECTOR_POD_IP}:4317
+            http:
+                endpoint: ${STRAWS_COLLECTOR_POD_IP}:4318
+processors:
+    transform/remove_attributes_1:
+        error_mode: ignore
```

Comment:
This is an invalid transformprocessor config, it will cause the collector not to start.